core/storage/app_feedback_report/gae_models.py

```diff
@@ -142,6 +152,7 @@ def create(
             entry_point_exploration_id, entry_point_subtopic_id,
             text_language_code, audio_language_code, android_report_info,
             web_report_info):
+        # type: (Text, datetime.datetime, Text, Text, Text, Optional[Text], Optional[int], Optional[Text], Text, Optional[Text], Optional[Text], Optional[Text], Optional[Text], Text, Text, Optional[Dict[Text, Any]], Optional[Dict[Text, Any]]) -> Text
```

Comment:
This is very difficult to read, can't this be split into multiple lines? Causing lint issues?

Reply:
The type annotations cannot be broken down into multiple lines.
There is one other way to do this:
```
def send_email(address,     # type: Union[str, List[str]]
               sender,      # type: str
               cc,          # type: Optional[List[str]]
               bcc,         # type: Optional[List[str]]
               subject='',
               body=None    # type: List[str]
               ):
    # type: (...) -> bool
```

What do you prefer?

Reply:
For example if you consider the `create()` function above (for which I made this comment) there are about 17 parameters, and to find out the type of any of these parameters from single line looks cumbersome (you have to count and map manually) - the new method you proposed would increase the number of lines, but I think that cost is justified by the readability it provides. @hardikkat24 Have you verified that this doesn't introduce any lint errors?

@seanlip @DubeySandeep Just to confirm - does the second method introduce any anti-pattern? I agree that to avoid confusion as to which method to use, there should be some criteria that helps a developer decide that - something like for `<= x` (=10 maybe) parameters method 1 (the current one used to define annotation in single line) should be used, else method 2. I am a bit skeptical here and wanted an opinion first. Thanks!

Reply:
Multiple lines seems fine to me. I would probably just break after the '(' rather than push everything to the right.

@DubeySandeep what do you think?

Also, will this change once we move to Python 3? What is the plan then?

Reply:
1. Are we going to make `#` look vertically oriented all the time? (This will make it hard to do new changes)
2. `# type: (...) -> bool` Are we going to have something in between `...` or is it just a syntaxt to hilight the usage of inline param type?
3. Can we enforce to only have this way to write param's type? [Just to have consistent pattern in ethc odebase and I think the new pattern looks more consistent to fronted type + I think after python 3 migration we are going to have the same structure, right?]

In general, I agree this would be a better way to present the type for params!

Reply:
@DubeySandeep 
1. I don't prefer making the type comments for this to be vertically oriented.
2. The `()` bracket generally contains the types of the function parameters, but here we have specified them in the lines itself, so `(...)` will be used.
3. I don't think breaking the function definition will look good for comparatively smaller functions. In python 3, we will have  syntax like this:
   ```
   def func(x: int, y: int, z: bool) -> bool:
      ...
   ```
   So here we are not breaking the function call.

Reply:
Split the current function into multiple lines